torchaudio/backend/sox_io_backend.py

```diff
@@ -1,4 +1,5 @@
-from typing import Tuple, Optional
+from typing import Tuple, Optional, Union
+from pathlib import Path
```

Comment:
Do we still need these changes?

Reply:
good catch! current update reflects these comments.